apps/developer-hub/next.config.js

```diff
@@ -0,0 +1,77 @@
+import createBundleAnalyzer from "@next/bundle-analyzer";
+import { createMDX } from "fumadocs-mdx/next";
+
+const withAnalyzer = createBundleAnalyzer({
+  enabled: process.env.ANALYZE === "true",
+});
+
+const config = {
+  reactStrictMode: true,
+  turbopack: {
+    resolveExtensions: [".mdx", ".tsx", ".ts", ".jsx", ".js", ".mjs", ".json"],
+    rules: {
+      "*.svg": {
+        loaders: ["@svgr/webpack"],
+        as: "*.js",
+      },
+    },
+  },
+
+  pageExtensions: ["ts", "tsx", "mdx"],
+
+  experimental: {
+    useCache: true,
```

Comment:
I don't think you need this experiment if you aren't using the `"use cache"` directive

Reply:
This has just been copied across from Insights. I didn't really change anything I wasn't 100% sure about. It looks like there's a few other comments on the next config that are probably for similar reasons. I'll compile all the requested config changes into a single commit rather than address them individually.

Reply:
no problem -- in this case we are in fact using the experimental caching in insights hub, so this is needed there, but I don't recommend using it unless you have a really good reason to and I don't think you will for dev hub